web/src/androidTest/kotlin/com/google/accompanist/web/DummyTests.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.swiperefresh
+
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/**
+ * Dummy tests to help with sharding: https://github.com/android/android-test/issues/973
+ */
+@RunWith(JUnit4::class)
+class DummyTests {
```

Comment:
Rename to be more inclusive https://developers.google.com/style/word-list?hl=en#dummy-variable

Reply:
Used throughout the project so will need updating elsewhere but will update this one for now